allennlp/commands/train.py

```diff
@@ -1,5 +1,5 @@
 """
-The ``train`` subcommand can be used to train a model.
```

Comment:
It's best not to mix in formatting cleanups with functional changes. Makes the diff harder to read.